mvcoverride.php

```diff
@@ -226,7 +226,7 @@ private function getOverrideFileInfo($includePath, $component, $filePath, $type
 			$baseName = basename($filePath);
 			$prefix = substr($baseName, 0, 5);
 
-			if (($app->isAdmin() && $prefix == 'admin') || (!$app->isAdmin() && $prefix != 'admin') )
+			if (($app->isAdmin() && $prefix == 'admin') || (!$app->isAdmin() && $prefix != 'admin') || !(!$app->isAdmin() && $prefix == 'admin'))
```

Comment:
yes but those two negate each other... so you could have it like this:
if (($app->isAdmin() && $prefix == 'admin') || (!$app->isAdmin()))
it is the same check as yours above